docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
This is a bit tricky. We have to set the theme statically, since this is hard-coded into the README. This logo is actually `xclim-logo-small.png`. I can make a dark version, though.